src/lib/Gen/xml/XMLWriter.cpp

```diff
@@ -381,6 +385,81 @@ writeGuide(
     closeTemplate(I.Template);
 }
 
+void
+XMLWriter::
+writeAlias(
+    AliasInfo const& I)
+{
+    tags_.open(aliasTagName, {
+        { "name", I.Name },
+        { I.Access },
+        { I.id }
+        });
+
+    writeSourceInfo(I);
+
+    writeJavadoc(I.javadoc);
+
+    if (I.AliasedSymbol)
+    {
+        Attributes nameAttrs = {};
+        nameAttrs.push({"name", toString(*I.AliasedSymbol)});
+        tags_.write("name", {}, nameAttrs);
+    }
+
+    tags_.close(aliasTagName);
+}
+
+void
+XMLWriter::
+    writeUsing(UsingInfo const& I)
+{
+    dom::String classStr;
+    switch (I.Class)
+    {
+    case UsingClass::Normal:
+        classStr = "using";
+        break;
+    case UsingClass::Typename:
+        classStr = "using typename";
+        break;
+    case UsingClass::Enum:
+        classStr = "using enum";
+        break;
+    case UsingClass::Namespace:
+        classStr = "using namespace";
+        break;
+    default:
+        MRDOCS_UNREACHABLE();
+    }
+
+    tags_.open(usingTagName, {
+        { I.Access },
+        { I.id },
+        { "class", classStr, ! classStr.empty() }
+    });
+
+    writeSourceInfo(I);
+
+    writeJavadoc(I.javadoc);
+
+    for (auto const& symbol : I.UsingSymbols)
+    {
+        Attributes attrs = {};
+        attrs.push({"id", toString(symbol)});
+        tags_.write("symbol", {}, attrs);
+    }
+
+    if (I.Qualifier)
+    {
+        Attributes nameAttrs = {};
```

Comment:
Same thing here... you can just use list-initialization to construct the `Attributes` parameter directly